### ISSUE TITLE
Excessive queued sounds cause applause sound to play past score screen

### ISSUE THREAD

**Issue by aerouk** [ppy/osu-stable-issues#13]
The title may be a bit wordy but I think that's the best way to put it. If you cause the gameplay stage to queue lots of sound samples (for example, stacking loads of spinners on top of each other), they will keep playing into the score screen.

If you leave the score screen before the sounds have finished playing, the applause sound from the skin will continue playing with no way to stop it except closing the game or changing/reloading the skin.

Here's a video example: [https://streamable.com/vtiut](https://streamable.com/vtiut)

**Comment by peppy** [comment]
I can understand how this would happen. Should be a relatively simple (hacky) fix.

**Comment by peppy** [comment]
Fixed next cutting dege.
